Gemfile

```diff
@@ -6,10 +6,12 @@ ruby file: ".ruby-version"
 gem "batch_api"
 gem "bcrypt", "~> 3.1.7"
 gem "clockwork"
-gem "devise", "~> 4.9.0"
+gem "devise"
 gem "devise_token_auth", "~> 1.2"
 gem "omniauth-azure-activedirectory-v2", "~> 2.1.0"
+gem "foundation-rails"
```

Comment:
i think foundation-rails, jquery-rails, and sass-rails should no longer be needed

Reply:
This should help us later when we move to Rails 8

Reply:
not sure why we would need it as we don't have a rails front end

Reply:
as in: removing them will help us later 😄

Reply:
thanks for clarifying. i was confused why they were re-added after i initially removed them (and thought Jonathan was making an argument for keeping them)